Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -29,6 +30,12 @@
     <Reference Include="System.Web" />
   </ItemGroup>
   
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+    <PackageReference Include="Microsoft.PowerShell.Commands.Utility">

```

Comment:
What is powershell commands utitlity used for?

Reply:
Powershell is not actually used by the driver. But .NET Core 2.0 has a vulnerability which upgrading Powershell to 6.1.0 is part of the fix: https://www.whitesourcesoftware.com/vulnerability-database/CVE-2018-8292